src/blocks/product-card.css

```diff
@@ -0,0 +1,128 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+}
+
+.card__product-image {
+  padding: 0 19px;
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding: 0 16px;
+  width: 166px;
```

Comment:
There is no point in setting width on most of these .card children (also height as height of the content is enough)

Reply:
I deleted this `width`.